### PR TITLE
Fix package

### PR DESCRIPTION
**Pull request by gaetanbrison**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,10 +22,9 @@
     test_suite="nose.collector",
     classifiers=[
         "License :: OSI Approved :: MIT License",
-        "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.7",
```
Comment (agramfort) [inline setup.py:25]: this means it only works only on 3.7
you should list it works with the different python 3 versions we test on the CI

File: setup.py [hunk 2]
```diff
@@ -22,10 +22,9 @@
...
     ],
-    install_requires=["pandas", "numpy", "scipy", "scikit-learn"],
+    install_requires=["pandas", "numpy", "scipy", "scikit-learn","arff2pandas","liac-arff"],
```
Comment (agramfort) [inline setup.py:27]: please run black on your code and respect pep8 rules

**Review by agramfort — COMMENTED** [review]
@gaetanbrison can you avoid opening PRs from the upstream repo to avoid having dead branches in the upstream repo?

also we mentioned the flake8 pass to be done when you find the time.

🙏 🙏 🙏

File: setup.py (hunk 1 above)
Comment (gaetanbrison) [reply setup.py:25]: yes

File: setup.py (hunk 2 above)
Comment (gaetanbrison) [reply setup.py:27]: sounds good

**Pull request merged**